### PR TITLE
[FLAPPY-1] Added basic html and js files

### PR DESCRIPTION
**Pull request by PawCyb9809**
Please review

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+let canv = document.querySelector('#Flappy');
+let ctx = cvs.getContext("2d");
```
Comment (teclamat) [inline main.js:2]: Those lines wont work, you try to access `canvas` with id `Flappy` before it loaded.  Second thing - try using `const` everywhere and change to `let` when necessary.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="pl">
```
Comment (teclamat) [inline index.html:2]: `lang="en"`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Flappy Bird Game</title>
+    <script src="main.js"></script>
+    <style>
+        #Flappy{
```
Comment (teclamat) [inline index.html:8]: Missing space before braces

File: index.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            border: 1px solid #949494;
+            display: block;
+            margin: 50px auto;
+        }
+    </style>
+</head>
+<body>
+    <canvas id="Flappy" width="300" height="530"></canvas>
+</body>
+</html>
```
Comment (teclamat) [inline index.html:18]: Missing empty line at theend of file

File: main.js (hunk 1 above)
Comment (teclamat) [inline main.js:2]: Missing empty line at theend of file

File: index.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <title>Flappy Bird Game</title>
+    <script src="main.js"></script>
+    <style>
+        #Flappy{
+            border: 1px solid #949494;
+            display: block;
+            margin: 50px auto;
```
Comment (teclamat) [inline index.html:11]: Top and bottom margin of `50px` might "bite" us later on

File: index.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <title>Flappy Bird Game</title>
+    <script src="main.js"></script>
+    <style>
+        #Flappy{
+            border: 1px solid #949494;
```
Comment (teclamat) [inline index.html:9]: Any particular reason you made border in color `#949494` instead of `gray` or `black`?

Comment (PawCyb9809) [reply index.html:9]: For no reason, I change to black.

**Pull request merged**